static/app/views/performance/queues/messageSamplesPanel.tsx

```diff
@@ -257,6 +256,90 @@ export function MessageConsumerSamplesPanel() {
   );
 }
 
+function MetricsRibbon({
+  type,
+  metrics,
+  isLoading,
+}: {
+  isLoading: boolean;
+  metrics: Partial<SpanMetricsResponse>[];
+  type: MessageActorType;
+}) {
+  switch (type) {
+    case MessageActorType.PRODUCER:
+      return <MetricsRibbonProducer metrics={metrics} isLoading={isLoading} />;
+    case MessageActorType.CONSUMER:
+    default:
+      return <MetricsRibbonConsumer metrics={metrics} isLoading={isLoading} />;
```

Comment:
I think `MetricsRibbonProducer` should be `ProducerMetricsRibbon`, otherwise it sounds like this component is a producer of something rather than a ribbon of something

Reply:
lol good point, updated the name!